packages/ListBoxWithTags/CHANGELOG.md

```diff
@@ -20,7 +20,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 
 - Support for `isDisabled`, `isReadOnly`, and `hasError` props. [@mikrotron](https://github.com/mikrotron)
-- Support fo `<ListBoxWithTags.A11y>` props collector. [@mikrotron](https://github.com/mikrotron)
+- Support for `<ListBoxWithTags.A11y>` props collector. [@mikrotron](https://github.com/mikrotron)
+- Support for `placeholder` prop. [@mikrotron](https://github.com/mikrotron)
```

Comment:
Not sure if we should be touching the `CHANGELOG` manually anymore? Seems like we can just rely on the `changeset` message?

cc @allison-c

Reply:
Yes, you're right